packages/picasso/src/PageHead/story/Steps.example.tsx

```diff
@@ -4,7 +4,9 @@ import { PageHead, Stepper } from '@toptal/picasso'
 const StepsExample = () => (
   <PageHead>
     <PageHead.Main>
-      <Stepper steps={['Label', 'Label', 'Label', 'Label', 'Label']} />
+      <Stepper
+        steps={['Label-1', 'Label-2', 'Label-3', 'Label-4', 'Label-5']}
```

Comment:
Just need to generate visual snapshot for this one 👍

Reply:
Maybe for better presentation?
```suggestion
        steps={['Label 1', 'Label 2', 'Label 3', 'Label 4', 'Label 5']}
```

Reply:
I will add the new visual snapshot and thanks for the suggestion 👍

Reply:
Maybe use some more real-world labels? maybe we can take from real projects?